src/headless/plugins/chat/model.js

```diff
@@ -731,6 +731,7 @@ const ChatBox = ModelWithContact.extend({
                 'id': origin_id,
                 'xmlns': Strophe.NS.FASTEN
             }).c('retract', {xmlns: Strophe.NS.RETRACT})
+            .c('fallback', {xmlns: Strophe.NS.FALLBACK}).c('body');
```

Comment:
@PawBud: You're adding the `body` element as a child of the `fallback` element. You need to call `.up()` first (like on line 729).

You also need to add text to the body, with the `.t` function.

For example: `.c('body').t('This person attempted to retract a previous message, but it's unsupported by your client.')`